### PR TITLE
feat: Pin dependencies and unify tsconfig

### PR DESCRIPTION
**Pull request by lalalune**
This PR unifies the tsconfig to all import from the core config as the root of their own tsconfig, as opposed to the tsconfig in monorepo root.

This is recommended by turbo to unify the tsc versions between the package versions and locally installed.

### CODE REVIEW

File: packages/plugin-node/src/services/llama.ts [hunk 1]
```diff
@@ -185,10 +185,34 @@ export class LlamaService extends Service {
...
+            );
+            return "./";
+        }
+
         this.modelPath = path.join(
             process.env.LLAMALOCAL_PATH?.trim() ?? "./",
             modelName
         );
+
+        console.log("*** MODEL PATH: ", this.modelPath);
```
Comment (cygaar) [inline packages/plugin-node/src/services/llama.ts:213]: do we need these console.logs?

File: eslint.config.mjs [hunk 2]
```diff
@@ -12,8 +12,7 @@ export default [
             parser: typescript,
             parserOptions: {
                 ecmaVersion: "latest",
-                sourceType: "module",
-                project: "./tsconfig.json", // Make sure your tsconfig includes @types/node
```
Comment (cygaar) [inline eslint.config.mjs:16]: how does eslint know where to look for the typescript config now?

Comment (lalalune) [reply eslint.config.mjs]: fixed

**Comment by fabianhug** [comment]
Does it make sense to pump the versions in the `package.json` definitions directly to `"version": "0.1.4-alpha.4",` instead of `alpha.3`?

File: package.json [hunk 3]
```diff
@@ -52,6 +52,7 @@
         "@coinbase/coinbase-sdk": "0.10.0",
         "amqplib": "0.10.5",
         "csv-parse": "5.6.0",
+        "eslint-plugin-vitest": "^0.5.4",
```
Comment (cygaar) [inline package.json:55]: this should probably be pinned

**Review by cygaar — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (fabianhug) [reply package.json:55]: +1

**Comment by fabianhug** [comment]
lgtm, good catch /w config @cygaar

**Pull request merged**